R/zchunk_L251.en_ssp_nonco2.R

```diff
@@ -1,27 +1,32 @@
 #' module_emissions_L251.en_ssp_nonco2
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces regional non-CO2 emissions coefficient data for SSPs 1/5, 2, and 3/4 as well as a GDP control.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L251.ctrl.delete}, \code{L251.ssp15_ef}, \code{L251.ssp2_ef}, \code{L251.ssp34_ef}, \code{L251.ssp15_ef_vin}, \code{L251.ssp2_ef_vin}, \code{L251.ssp34_ef_vin}. The corresponding file in the
 #' original data system was \code{L251.en_ssp_nonco2.R} (emissions level2).
-#' @details Describe in detail what this chunk does.
+#' @details This section takes in the non-CO2 emissions factors for SSP 1/5, 2, and 3/4 across sectors.
+#' First, create data that spans the years 2010-2100 in five year increments by interpolation of input data.
+#' Next, add emissions controls for future years of vintaged technologies for SSP emission factors.
+#' Then, add columns that have regional SO2 emission species.
+#' A GDP control of regional non-CO2 emissions in all regions is also created.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_emissions_L251.en_ssp_nonco2_DISABLED <- function(command, ...) {
+#' @author CDL May 2017
+
+module_emissions_L251.en_ssp_nonco2 <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
-    return(c(FILE = "common/GCAM_region_names",
-             FILE = "emissions/A_regions",
+    return(c(FILE = "emissions/A_regions",
+             FILE = "common/GCAM_region_names",
```

Comment:
Could you move any emission mapping files from the emissions directory into the emissions/mappings directory?